homeassistant/components/sensor/upnp.py

```diff
@@ -100,6 +105,17 @@ def _upnp_remove_sensor(self, device):
 
         self.hass.async_create_task(self.async_remove())
 
+    @property
+    def device_info(self):
+        """Get device info."""
+        return {
+            'identifiers': {
+                (DOMAIN_UPNP, self.unique_id)
+            },
+            'name': self.name,
+            'via_hub': (DOMAIN_UPNP, self._device.udn),
```

Comment:
I see that you register a via_hub here but that device is not created anywhere ?

Reply:
It is created in the [`upnp` component itself](https://github.com/home-assistant/home-assistant/pull/18565/files/b992771b62a1ba474338e1f3167ae1f9a472ee9f#diff-eb17263c144de90ae5fccabcf9cafb05R151)